src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,23 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    enum Color {
+        YELLOW,
+        GREEN,
+        BLACK,
+        BLUE,
+        PINK,
+        PURPLE,
+        RED,
+        WHITE,
+        ORANGE
+    }
+
+    private static final int COLOR_COUNT = 9;
```

Comment:
this code is poorly supported, if you add a new color you will need to change this number. Let's remove it and use Color.values().length